src/main/java/com/microsoft/azure/springcloudplayground/util/TelemetryProxy.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.springcloudplayground.util;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+import org.springframework.lang.NonNull;
+import org.springframework.lang.Nullable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TelemetryProxy {
+
+    private static final String PROJECT_PROPERTY_FILE = "/META-INF/project.properties";
+
+    private final TelemetryClient client;
+    private final PropertyLoader loader;
+
+    public TelemetryProxy() {
+        this.client = new TelemetryClient();
+        this.loader = new PropertyLoader(PROJECT_PROPERTY_FILE);
```

Comment:
loader is just used once. if it's big better pass in the value when constructing the class

Reply:
Yes, we can make it static here.